pyfr/util.py

```diff
@@ -168,3 +169,20 @@ def match_paired_paren(delim, n=5):
     rgt = rf'\{close}{ocset}*?)*?'
 
     return lft*n + mid + rgt*n
+
+def file_path_gen(basedir, basename):
```

Comment:
Switch to the newer version I wrote which does not require a `next()` after send.